src/main/java/Calculator.java

```diff
@@ -0,0 +1,79 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Calculator {
+    Scanner scanner = new Scanner(System.in);
+    private final String ANSI_RED = "\u001B[31m";
+    private final String ANSI_RESET = "\u001B[0m";
+    private double tempArg;
+
+    public void numbersValidator() {
+        double input;
+        boolean inputIsValid = false;
+
+        do {
+            try {
+                input = scanner.nextDouble();
+                tempArg = input;
+                inputIsValid = true;
+            } catch (InputMismatchException e) {
+                System.out.print(ANSI_RED + "Wrong input! Try Again: " + ANSI_RESET);
+                scanner.next();
+            }
+        } while (!inputIsValid);
+    }
+
+    public double doCalc(double firstArg, double secondArg, String operator) {
+        double result = Double.NaN;
+        try {
+            switch (operator) {
+                case "+" -> result = firstArg + secondArg;
+                case "-" -> result = firstArg - secondArg;
+                case "*" -> result = firstArg * secondArg;
+                case "/" -> {
+                    if (secondArg != 0) {
+                        result = firstArg / secondArg;
+                    } else {
+                        throw new ArithmeticException(ANSI_RED + "Dividing by zero!" + ANSI_RESET);
+                    }
+                }
+                default -> System.out.println(ANSI_RED + "Wrong option!" + ANSI_RESET);
+            }
+        } catch (ArithmeticException e) {
+            System.out.println(e.getMessage());
+        }
+        return result;
+    }
+
+    public void menu() {
+        while (true) {
+            System.out.print("Choose option (+, -, *, /) or \"exit\" for ending program: ");
+            String operator = scanner.next().toLowerCase();
```

Comment:
I don't know if it was a good decision to input option first before the number input, but I think that it's better than inputting the number and then typing 'exit.' Please correct me if I'm wrong.